lvm.go

```diff
@@ -51,7 +51,7 @@ type VolumeManager interface {
 	RemoveLV(lvName string) error
 
 	// ExtendLV expands the LV to the requested new size.
-	ExtendLV(newSize uint64) error
+	ExtendLV(lvName string, newSize uint64) error
```

Comment:
What did you expect to pass here as lvName ?  I think that this needs `vgName string` as its first param.

Also, RemoveLV and HasLV also need ammending.

Reply:
I thought from LV we can detect the Vg and then call correct extend method. From a developer point of view he needs to extend the LV he just created. Would that make sense? We can also take the vg and lv name if that simplifies implementation.

Reply:
sorry... i should have explained more.  lv name is not system unique. so one way or another you have to give the volume group.

    $ vgcreate vg0 /dev/vda
    $ vgcreate vg1 /dev/vdb
    $ lvcreate --size=100MiB --name=mylv vg0
    $ lvcreate --size=100MiB --name=mylv vg1

Now I have two logical volumes named 'mylv'.